Source/Scene/Scene.js

```diff
@@ -702,6 +718,30 @@ define([
         },
 
         /**
+         * Gets the event that will be raised at when the camera starts to move.
+         * @memberof Scene.prototype
+         * @type {Event}
+         * @readonly
+         */
+        cameraMoveStart : {
```

Comment:
Is it possible/reasonable to put these events on the camera, but fire them from the scene?  Don't you think this will be a better API from the users' perspective?